main.go

```diff
@@ -351,6 +363,24 @@ func main() {
 
 }
 
+// regexp that parses Cloudformation paramter key-value pair: https://regex101.com/r/hruxlg/3
+var paramRe = regexp.MustCompile(`(?:ParameterKey)=("(?:\\.|[^"\\]+)*"|(?:\\.|[^, "\\]+)*),(?:ParameterValue)=("(?:\\.|[^"\\]+)*"|(?:\\.|[^ ,"\\]+)*)`)
```

Comment:
Not a big fan of regex. You could have done just a traditional parsing. But this is fine